src/store/index.js

```diff
@@ -10,7 +10,8 @@ export default new Vuex.Store({
   state: {
     board: [],
     posts: [],
-    events: []
+    events: [],
```

Comment:
What data is being added to the Vuex store here and when is it accessed? 

Unless I'm mistaken, I don't think we need this, we can just send a post request to the correct endpoint.